@LinProp/LinProp.m

```diff
@@ -1354,6 +1354,11 @@ function display(obj)
             z = x*inv(y);
         end
         function z = mtimes(x,y)
+            if isscalar(x) || isscalar(y)
+                z = times(x, y);
+                return
```

Comment:
I don't like the `return` in the if case. I like more if you would use if else end and my code in the else case.